### ISSUE TITLE
Customizing tests 

### ISSUE THREAD

**Issue by bitcity** [pennersr/django-allauth#1236]
If we override the templates, some tests that check for specific words in templates might fail. What's a good way to run allauth tests in our own project setup ?

**Comment by pennersr** [comment]
Test code needs to change then. Got any list of failing tests?

**Comment by bitcity** [comment]
```
    def test_password_forgotten_username_hint(self):
        self._request_new_password()
        body = mail.outbox[0].body
        assert 'username' in body
```

This assumes that `username` is mentioned in e-mail.

**Comment by pennersr** [comment]
This specific issue is addressed in 52a80d6, but I am aware that there is probably more work to do in this area.

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/1236
